gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/OrderabilityTest.java

```diff
@@ -177,6 +178,7 @@ public void g_E_properties_order_value() {
      * Mixed type values including list, set, map, uuid, date, boolean, numeric, string, null.
      */
     @Test
+    @Ignore // todo: re-enable after datetime serialization is implemented
```

Comment:
Can you clarify what is left to do for datetime serialization? I thought that this PR made this change so these tests shouldn't be disabled.

Reply:
That's correct, I missed these tests, I'll update it.